static/assets/menu.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
V případě (minimálně) této ikony a `close` ikony by bylo vhodné umístit SVG do složky `components/icons` a [převést SVG na React komponentu](https://react-svgr.com/playground/). 

Toto by bylo ideální i v případě loga Česko.Digital, ale nejsem si jistý, zda to bude fungovat v kombinaci s `background-image`.